CONTRIBUTING.md

```diff
@@ -1,34 +1,48 @@
-## 🙏 Welcome
+## ✨ Welcome
```

Comment:
Not sure who put that there but the folded hands is how we say welcome ([नमस्ते](https://en.wikipedia.org/wiki/Namaste)) in India.

Reply:
That is definitely an improvement.